app/serializers/user_serializer.rb

```diff
@@ -0,0 +1,26 @@
+# == Schema Information
+#
+# Table name: users
+#
+#  id                     :integer          not null, primary key
+#  email                  :string(255)      default(""), not null
+#  encrypted_password     :string(255)      default(""), not null
+#  reset_password_token   :string(255)
+#  reset_password_sent_at :datetime
+#  remember_created_at    :datetime
+#  sign_in_count          :integer          default(0), not null
+#  current_sign_in_at     :datetime
+#  last_sign_in_at        :datetime
+#  current_sign_in_ip     :string(255)
+#  last_sign_in_ip        :string(255)
+#  failed_attempts        :integer          default(0), not null
+#  unlock_token           :string(255)
+#  locked_at              :datetime
+#  created_at             :datetime
+#  updated_at             :datetime
+#  full_name              :string(255)      not null
+#
+
+class UserSerializer < ActiveModel::Serializer
```

Comment:
@yunixon because jbuilder is much more flexible. I tried other solutions in different projects and found that at some point you meet some limits.